### ISSUE TITLE
Module should not be called, Rails as its shaddows Rails itself.

### ISSUE THREAD

**Issue by nbrien** [statpro/devextreme-rails#24]
https://github.com/statpro/devextreme-rails/blob/f4ace3a474d5cedcf34814ef09ee4d9a1d5abcdd/lib/devextreme/rails.rb#L4

```NoMethodError: undefined method `logger' for Devextreme::Rails:Module
from /home/ubuntu/code/robusta/vendor/cache/devextreme-rails-76f535b001d6/lib/data_table.rb:252:in `block in apply_sorting'
```

Or we need to make sure that ::Rails is added below.. But this might only solve the few instances i have found

![image](https://user-images.githubusercontent.com/1502623/59592270-a022ef80-90e7-11e9-9ee4-4aa45b298319.png)

**Comment by WillemOpperman** [comment]
#25 Pull created

**Comment by ChristopherRollins** [comment]
I am closing this issue, as it has been resolved in pull #25. Therefore, this has been fixed since version [19.1.3.7](https://github.com/statpro/devextreme-rails/releases/tag/19.1.3.7).
